test/test-pages/gcba/platform-client.ts

```diff
@@ -0,0 +1,170 @@
+import { getCurrentUrlParams, getInputValue } from './utils';
+const client = (window as any).platformClient.ApiClient.instance;
+const persitentName = 'sdk_test';
+
+(function () {
+  function initClientEnvironment (environment) {
+    client.setPersistSettings(true, persitentName);
+    client.setEnvironment((window as any).environments[environment].uri);
+  }
+
+  function setAuthTextVisible (visible) {
+    const style = visible ? 'visible' : 'hidden';
+    document.getElementById('auth-text').style.visibility = style;
+  }
+
+  function signout () {
+    const environment: any = getInputValue('environment');
+    initClientEnvironment(environment);
+
+    const host = window.location.host;
+    let redirectUri = 'https://';
+
+    if (host === 'localhost:8443') {
+      redirectUri += host;
+    } else {
+      redirectUri += environment.uri;
+    }
+
+    redirectUri += window.location.pathname;
+    console.debug('Signing out with redirect of: ' + redirectUri);
+
+    client.logout(redirectUri);
+  }
+
+  function authenticate () {
+    const environment = getInputValue('environment');
+    const clientId = (window as any).environments[environment].clientId;
+    initClientEnvironment(environment);
+
+    client.loginImplicitGrant(clientId, window.location.href)
+      .then(() => {
+        const authInfo = JSON.parse(window.localStorage.getItem(`${persitentName}_auth_data`));
+        (client as any).authentications['PureCloud Auth'].accessToken = authInfo.accessToken;
+
+        (window as any).conversationsAPI = new (window as any).platformClientConversationsApi();
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9580329)
```suggestion
        (window as unknown).conversationsAPI = new (window as any).platformClientConversationsApi();
```